module/TaskManagement/src/TaskManagement/Controller/TasksController.php

```diff
@@ -50,7 +50,7 @@ public function __construct(TaskService $taskService, StreamService $streamServi
 		$this->streamService = $streamService;		
 		$this->acl = $acl;

```

Comment:
in questo caso il servizio acl non è usato nel controller per chiamare isAllowed, ma serve nella costruzione di TaskJsonModel